server.md

```diff
@@ -1,22 +1,31 @@
 # Server Conventions
 
-The following outlines our server conventions as it pertains to exported server
-interfaces/types:
+Edge uses a micro-services architecture with small codebases each dedicated to a specific purpose (login, file sync, exchange rates, and so forth). Each server lives in its own git repository with a name like "edge-purpose-server". Each server should have "private": true in its package.json file, since server code does not belong on NPM.
```

Comment:
We lost the back ticks on the `"private": true` bit, as well as some others places. Not sure if this is worth fixing.